### PR TITLE
Add tests for memb APIs

### PR DESCRIPTION
**Pull request by yatch**
This PR introduces simple tests for `memb` APIs. In addition, docstring in `memb.h` is updated.

### CODE REVIEW

File: tests/06-script-base/code-test-memb/contiki.h [hunk 1]
```diff
@@ -0,0 +1,8 @@
+/*
+ * When memb.h is "included", contiki.h is also loaded as a result
+ * since it is a dependency of cc.h, which memb.h needs for
+ * CC_CONCAT() marcro.
```
Comment (evdenis) [inline tests/06-script-base/code-test-memb/contiki.h]: s/marcro/macro/

Comment (yatch) [reply tests/06-script-base/code-test-memb/contiki.h]: Oh, thanks!

**Comment by yatch** [comment]
This branch will be updated once PR #1003 gets merged, otherwise the "double-free" test will fail.

**Comment by joakimeriksson** [comment]
Now #1003 is merged. Will you do an update - I can do a review when you have made the update.

**Comment by yatch** [comment]
Thank you; I've rebased and updated the branch.

**Comment by yatch** [comment]
The test of `tun-rpl-br` failed... I don't think the changes by this PR caused the failure. I will re-run the test...

Here is an excerpt of the test result:

```
==== 02-border-router-cooja-tsch.tunslip6.log ====
make[1]: Entering directory '/home/user/contiki-ng/examples/rpl-border-router'
sudo ../../tools/serial-io/tunslip6 -a 127.0.0.1 fd00::1/64
slip connected to ``127.0.0.1:60001''
opened tun device ``/dev/tun0''
ifconfig tun0 inet `hostname` mtu 1500 up
ifconfig tun0 add fd00::1/64
ifconfig tun0 add fe80::0:0:0:1/64
ifconfig tun0
tun0      Link encap:UNSPEC  HWaddr 00-00-00-00-00-00-00-00-00-00-00-00-00-00-00-00  
          inet addr:172.17.0.2  P-t-P:172.17.0.2  Mask:255.255.255.255
          inet6 addr: fd00::1/64 Scope:Global
          inet6 addr: fe80::1/64 Scope:Link
          UP POINTOPOINT RUNNING NOARP MULTICAST  MTU:1500  Metric:1
          RX packets:0 errors:0 dropped:0 overruns:0 frame:0
          TX packets:0 errors:0 dropped:0 overruns:0 carrier:0
          collisions:0 txqueuelen:500 
          RX bytes:0 (0.0 B)  TX bytes:0 (0.0 B)
[INFO: BR        ] Waiting for prefix
*** Address:fd00::1 => fd00:0000:0000:0000
[INFO: BR        ] Waiting for prefix
[INFO: BR        ] Server IPv6 addresses:
[INFO: BR        ]   fd00::201:1:1:1
[INFO: BR        ]   fe80::201:1:1:1
tunslip6: serial_to_tun: read: Resource temporarily unavailable
ifconfig tun0 down
netstat -nr | awk '{ if ($2 == "tun0") print "route delete -net "$1; }' | sh
==== 02-border-router-cooja-tsch.scriptlog ====
PING fd00::204:4:4:4(fd00::204:4:4:4) 56 data bytes
64 bytes from fd00::204:4:4:4: icmp_seq=1 ttl=61 time=415 ms
64 bytes from fd00::204:4:4:4: icmp_seq=2 ttl=61 time=394 ms
64 bytes from fd00::204:4:4:4: icmp_seq=3 ttl=61 time=443 ms
64 bytes from fd00::204:4:4:4: icmp_seq=4 ttl=61 time=423 ms
--- fd00::204:4:4:4 ping statistics ---
5 packets transmitted, 4 received, 20% packet loss, time 4003ms
rtt min/avg/max/mdev = 394.500/419.490/443.685/17.628 ms
02-border-router-cooja-tsch      TEST FAIL

(snip)

========== Summary ==========
01-border-router-cooja           TEST OK
02-border-router-cooja-tsch      TEST FAIL
03-border-router-sky             TEST OK
04-border-router-traceroute      TEST OK
05-native-ping                   TEST OK
06-native-coap                   TEST OK      2/2
07-native-border-router-cooja    TEST OK
08-border-router-cooja-frag      TEST OK
09-native-border-router-cooja-frag TEST OK
```

**Comment by joakimeriksson** [comment]
No, this is not caused by the PR. It is an issue that needs fixing... Now and then it fails but typically a re-run works fine.

**Comment by yatch** [comment]
@joakimeriksson Oh, OK; thanks!

>  It is an issue that needs fixing... Now and then it fails but typically a re-run works fine.

**Pull request merged**